### PR TITLE
DOCS-1310: Add do something docs

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/services/slam/_index.md [hunk 1]
```diff
@@ -152,27 +152,6 @@ pcd_map = await slam_svc.get_point_cloud_map()
 Get the internal state of the SLAM algorithm required to continue mapping/localization.
 
 {{< tabs >}}
```
Comment (npentrel) [inline docs/services/slam/_index.md:154]: reordering since we generally have python first

Comment (sguequierre) [reply docs/services/slam/_index.md:154]: @npentrel Could you reorder the first methods on this page too? It looks like a few still have python second

**Review by sguequierre — APPROVED** [review]
LGTM one comment

File: docs/services/slam/_index.md (hunk 1 above)
Comment (npentrel) [reply docs/services/slam/_index.md:154]: done

**Pull request merged**